webapp/src/locales/de.json

```diff
@@ -2,13 +2,14 @@
     "menu": {
         "LiveView": "Live-Ansicht",
         "Settings": "Einstellungen",
-        "NetworkSettings": "Netzwerk",
-        "NTPSettings": "NTP",
-        "MQTTSettings": "MQTT",
-        "InverterSettings": "Wechselrichter",
-        "SecuritySettings": "Sicherheit",
-        "DTUSettings": "DTU",
-        "DeviceManager": "Hardware",
+        "NetworkSettings": "Netzwerk Einstellungen",
+        "NTPSettings": "NTP Einstellungen",
+        "MQTTSettings": "MQTT Einstellungen",
+        "WireguardSettings" : "Wireguard Einstellungen",
+        "InverterSettings": "Wechselrichter Einstellungen",
+        "SecuritySettings": "Sicherheitseinstellungen",
+        "DTUSettings": "DTU Einstellungen",
+        "DeviceManager": "Geräte-Manager",
```

Comment:
Einstellungen is already the title of the Settings menü. Do not repeat it in every entry.